### PR TITLE
Implement image to pdf with embeded text functionality

### PR DESCRIPTION
**Pull request by mskrip**

### CODE REVIEW

File: src/pyocr/libtesseract/__init__.py [hunk 1]
```diff
@@ -159,6 +159,29 @@ def image_to_string(image, lang=None, builder=None):
     return builder.get_output()
 
 
+def image_to_pdf(image, output_file, lang=None, input_file="stdin",
```
Comment (Lubo) [inline src/pyocr/libtesseract/__init__.py]: Perhaps you should comment on why there's `input_file` and why it's necessary or not.

Comment (mskrip) [reply src/pyocr/libtesseract/__init__.py]: Explained it in d57baae

File: src/pyocr/libtesseract/__init__.py [hunk 2]
```diff
@@ -159,6 +159,44 @@ def image_to_string(image, lang=None, builder=None):
...
+            output_file: path to the file that will be created, `.pdf` extension should not be
+                specified
+            lang: three letter language code see tesseract. Defaults to None. See
+                https://github.com/tesseract-ocr/tesseract/blob/master/doc/tesseract.1.asc#languages
+            input_file: path to the image file that should be beneath the text in output pdf.
+                If not specified (stdin, incorrect file) output pdf is correct but tesseract
+                writes some errors about not being able to open the file. Defaults to stdin.
+            tessdata_dir: path to tessdata directory. Defaults to /usr/share/tessdata
+            textonly: create pdf with only one invisible text layer. Defaults to False.
+    '''
```
Comment (Lubo) [inline src/pyocr/libtesseract/__init__.py]: The first level of indentation in this docstring is unnecessary. See [PEP 257](https://www.python.org/dev/peps/pep-0257/).

File: src/pyocr/libtesseract/__init__.py [hunk 3]
```diff
@@ -159,6 +159,44 @@ def image_to_string(image, lang=None, builder=None):
...
+                 tessdata_dir="/usr/share/tessdata", textonly=False):
+    '''
+        Creates pdf file with embeded text based on OCR from an image
+
+        Args:
+            image: image to be converted
+            output_file: path to the file that will be created, `.pdf` extension should not be
+                specified
+            lang: three letter language code see tesseract. Defaults to None. See
+                https://github.com/tesseract-ocr/tesseract/blob/master/doc/tesseract.1.asc#languages
```
Comment (Lubo) [inline src/pyocr/libtesseract/__init__.py]: This description should be rephrased.

Comment (mskrip) [reply src/pyocr/libtesseract/__init__.py]: Done in 053020c

File: src/pyocr/libtesseract/__init__.py (hunk 2 above)
Comment (mskrip) [reply src/pyocr/libtesseract/__init__.py]: Fixed in 8bd18b5

**Comment by Lubo** [comment]
Squash the commits, please.

**Comment by mskrip** [comment]
Squashed

**Pull request merged**